changelogs/fragments/63-refactor-login-port-check.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- refactor perform_checks function and move login_port check to module_utils/mysql.py (https://github.com/ansible-collections/community.proxysql/pull/63)
```

Comment:
```suggestion
- refactor ``perform_checks`` function and move ``login_port`` check to ``module_utils/mysql.py`` (https://github.com/ansible-collections/community.proxysql/pull/63).
```

Reply:
the dot is necessary according to the Ansible doc, other things are to highlight and are not necessary